test-integration/user.test.ts

```diff
@@ -199,6 +199,21 @@ describe('User class', () => {
         })
     );
 
+    test(
+        "checkPassword() throws error if password doesn't match",
+        catchAndPrintErrors(async () => {
+            const { user, password } = await createRandomID();
+
+            await user.login(await user.getUsername(), password);
+
+            await expect(user.checkPassword(password)).resolves.toBeTruthy();
+            await expect(user.checkPassword('Testing123!@')).rejects.toThrowError(SdkErrors.PasswordInValid);
+            await expect(user.checkPassword('password')).rejects.toThrowError(SdkErrors.PasswordFormatInvalid);
+
+            await user.logout();
+        })
+    );
+
```

Comment:
this is good but we also need a success case so one more test that returns true if password matches 👍

Reply:
isn't it already on line 209?

Reply:
no because this function was newly made by you so we want to test the success case